cse/lang/NltkStemmer.py

```diff
@@ -0,0 +1,27 @@
+from cse.lang.PreprocessorStep import PreprocessorStep
+
+class NltkStemmer(PreprocessorStep):
+
+    __stemmerType = None
+
+    def __init__(self, stemmerType):
+        self.__stemmerType = stemmerType
+
+
+    @staticmethod
+    def porter():
+        from nltk.stem.porter import PorterStemmer
+        return PorterStemmer()
+
+    @staticmethod
+    def porter2():
```

Comment:
As far as I know you have to download the language set in order to use it. Do we need to add something in the setup process of our developing box?

Reply:
Yeah, you're right. I added it to the `installDependencies` script, but during merging with your changes it got lost ...